bigbluebutton-html5/imports/api/2.0/cursor/server/handlers/cursorUpdate.js

```diff
@@ -0,0 +1,15 @@
+import Logger from '/imports/startup/server/logger';
+import { check } from 'meteor/check';
+import updateCursor from '../modifiers/updateCursor';
+
+export default function handleCursorUpdate({ header, body }) {  
+  const meetingId = header.meetingId;
```

Comment:
For Cursor and Whiteboard we (html5) will work on extracting the same info we used in 1.1 from the 2.0 messages. Once we recover the 1.1 functionality for the whiteboard we will do another iteration with adding multi-user/cursor support